tools/sigtest/src/main/java/com/sun/tdk/signaturetest/core/JDKExclude.java

```diff
@@ -27,18 +27,30 @@
 
 package com.sun.tdk.signaturetest.core;
 
+import com.sun.tdk.signaturetest.util.CommandLineParser;
+
 /**
  * JDKExclude represents the JDK classes to be excluded from signature testing.
  *
  * @author Scott Marlow
  */
-public interface JDKExclude {
+public class JDKExclude {
+
+    private static PackageGroup excludedJdkClasses = new PackageGroup(true);
+
+    public static void enable() {
+        excludedJdkClasses.addPackages(new String[] {"java", "javax.transaction.xa" });
```

Comment:
Should we also ignore things like `javax.sql`?

Reply:
Good point, I looked at the current https://github.com/jakartaee/platform-tck/blob/master/src/com/sun/ts/tests/signaturetest/signature-repository/jakarta.persistence.sig#L1840 file and do see that we should include javax.sql and others as well.

Perhaps we should exclude all of javax since all of the classes that we care about checking for (super-setting/sub-setting) changes should be in the `jakarta` package.

For reference, output of `grep javax` in signaturetest/signature-repository folder is attached to [javax.txt](https://github.com/eclipse-ee4j/jakartaee-tck-tools/files/14906988/javax.txt)

Reply:
^^ I think it would be appropriate to ignore all `javax` packages assuming that they are part of the JDK since this artifact is part of `jakartaee` tools.  But maybe put a big warning somewhere in the README that we will ignore `java, and javax` just in case :)